packages/website/components/TableOfContent.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+import React from 'react';
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { List, TextLink, Subheading } from '@contentful/f36-components';
+
+const styles = {
+  root: css({
+    display: 'block',
+    background: 'white',
+    marginBottom: tokens.spacingL,
+    padding: tokens.spacingL,
+    marginTop: `calc(-1 * ${tokens.spacingL})`,
+    '@media (min-width: 1400px)': {
+      width: '350px',
+      marginLeft: '50px',
+      marginRight: '-100px',
+      display: 'inline-block',
+      background: 'white',
+      float: 'right',
```

Comment:
float ❤️
At first glance on the website, I couldn't get how you did that magic with layout. But then I found that lovely `float` 😄 

I guess, in near future, we gonna make this TOC sticky, so it stays with scroll, right?

Reply:
yeah, we can make it sticky later

Reply:
old school tricks 👴🏼 🪄

Reply:
WOW 😮